varya/README.md

```diff
@@ -0,0 +1,127 @@
+# Varya
+
+Varya is a styling system for quickly creating Gutenberg-ready themes for WordPress.
+
+## What does it do?
+
+When you reduce a theme design down to a set of systematic design decisions, you end up with something called a _Style Guide_. The Varya system works by taking the rules of a Style Guide and expressing them through carefully placed variables or _design tokens_ that influence the appearance of a WordPress site. It currently unifyies the overall of Gutenberg Blocks, the theme Header + Footer areas, WooCommerce, Jetpack and more. It also syncs styles between the editor and the frontend so that you don’t need to hand-write styles for both. This greatly speeds up the Gutenberg theme development process and reduces the amount of manual styling that typically goes into developing a theme. 
+
+### What controls does the system come with?
+
+  - **Fonts** - Font-family, size, weight, and line-height rules. 
+  - **Colors** - Primary, secondary, background, foreground and border colors. 
+  - **Spacing** - Sets an 8px vertical rhythm between all blocks and major components. Adds general spacing rules for blocks and components. Adds utility spacing classes for negative margins.
+  - **Responsive Logic** - Unifies the responsive behavior across various Blocks and Components to simplify behaviors.
+
+## How does it work?
+
+The system itself lives in the `/varya/sass` directory as a collection of Sass partials broken up by scope and hierarchy. The partials get compiled down to singular CSS files that live in the main Varya directory and cascade downward like so:
+
+**Frontend** 
+- `/varya/variables.css`
+- `/varya/style.css`
+
+**Editor** 
+- `/varya/variables-editor.css`
+- `/varya/style-editor.css`
+
+**Customizer**
+- `/varya/variables.css`
```

Comment:
Is there a reason the customizer variables and frontend variables are contained in the same partial?

Reply:
Woops! Yeah, that was left over from when Varya still had customizer color options. Now that those are gone, I can remove this section.